pkgs/by-name/pa/pasco/package.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "pasco";
+  version = "20040505_1";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/fast/Pasco/Pasco%20v${finalAttrs.version}/pasco_${finalAttrs.version}.tar.gz";
+    hash = "sha256-o7jue+lgVxQQvFZOzJMGd1WihlD7Nb+1WaSutq9vaGg=";
+  };
+
+  patches = [
+    ./pasco.patch
+  ];
+
+  makeFlags = [
+    "-C src"
+  ];
+
+  postPatch = ''
+    substituteInPlace src/Makefile \
+      --replace gcc cc
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp bin/pasco $out/bin
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Examine the contents of Internet Explorer's cache files for forensic purposes";
+    mainProgram = "pasco";
+    homepage = "https://sourceforge.net/projects/fast/files/Pasco/";
+    maintainers = with maintainers; [ d3vil0p3r ];
+    platforms = platforms.unix;
+    license = with licenses; [ free ];
```

```suggestion
    license = with licenses; [ free ];
```
https://salsa.debian.org/pkg-security-team/pasco/-/blob/debian/master/src/pasco.c?ref_type=heads
think this is a bsd3 -- looks like: https://opensource.org/license/BSD-3-clause/

---

Thank you. Is there a reason when I save the package.nix code by VSCodium and I save and push the change, the EditorConfig check produces an error related `Wrong new line` at the end despite I didn't put any new line at the end of the file?

---

https://github.com/NixOS/nixpkgs/blob/master/.editorconfig says that the last line of the file must end with a new line -- maybe your editor isn't adding them? https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files